app.R

```diff
@@ -100,8 +100,18 @@ FNimg<-readPNG('./www/FN.png')
 ui <- navbarPage(title = "MetaDTA: Diagnostic Test Accuracy Meta-analysis",
                   
                  #Set up google analytics 
-                 header = singleton(tags$head(includeScript("google_analytics.js"))),
-          
+                 header = singleton(
+                   tags$head(
+                     includeScript("google_analytics.js"),
+                     includeHTML("www/favicon/favicon.html"),
+                     tags$meta(name="description", content="An online interactive application for conducting meta-analysis of diagnostic test accuracy studies"),
+                     tags$meta(name="keywords", content="MetaDTA, DTA, Diagnostic, Test, Accuracy, Meta, Analysis, App"),
+                     tags$meta(property="og:title", content="MetaDTA: Diagnostic Test Accuracy Meta-analysis: V2.0.5"),
+                     tags$meta(property="og:description", content="An online interactive application for conducting meta-analysis of diagnostic test accuracy studies"),
+                     tags$meta(property="og:image", content="roc_curve.png")
```

Comment:
Upon checking this url needs to be the absolute url e.g.: https://crsu.shinyapps.io/dta_ma/roc.png

Though it might be better linking to the image on GH (as the image will not load if the instance needs to startup).
https://raw.githubusercontent.com/CRSU-Apps/MetaDTA/main/www/roc_curve.png